content/cloud/cloud-plugin.md

```diff
@@ -0,0 +1,363 @@
+title = "Cloud Command Reference"
+date = "2023-06-07T22:41:02.478752Z"
+template = "cloud_main"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/cloud-plugin.md"
+
+---
+- [Spin Cloud Command](#cloud)
+  - [Deploy](#deploy)
+  - [Login](#login)
+  - [Variables](#variables)
+    - [Set (Variables)](#set-variables)
+    - [Delete (Variables)](#delete-variables)
+    - [List (Variables)](#list-variables)
+  - [Subcommands Stability Table](#subcommands-stability-table)
+
+## Spin Cloud Command
+
+Fermyon provides a []`cloud` plugin](https://github.com/fermyon/cloud-plugin) for the Spin CLI for you to manage Spin applications in Fermyon Cloud. This page documents the `spin cloud` command. Specifically, all of the available options and subcommands. For more information on subcommand stability, see the [subcommands stability table](#subcommand-stability-table). You can reproduce the Spin Cloud command documentation on your machine by using the `--help` flag. For example:
+
+{{ tabs "cloud-plugin-version" }}
+
+{{ startTab "v0.1.0"}}
```

Comment:
As 0.1.1 will be the 'current' version at Spin 1.3 (which is itself the first time users can run `cloud` as a plugin), I'd suggest dropping the 0.1.0 tab.  Not a blocker if other folks want to keep it though!